node-backend/app/controllers/loginController.js

```diff
@@ -12,8 +13,8 @@ module.exports.login = async (req, res) => {
   let orgName;
   let orgId;
   let expTime;
-  const date = new Date();
-  const epoch = Math.round(date.getTime() / 1000);
+  let epoch = moment().valueOf();
```

Comment:
use moment.utc().unix() no need of any /1000 calculation. I guess you didn't read moment.js documentation